Packs/CommonScripts/Scripts/ConvertTimezoneFromUTC/ConvertTimezoneFromUTC_test.py

```diff
@@ -0,0 +1,36 @@
+from ConvertTimezoneFromUTC import convert_UTC_Timezone_command
+from ConvertTimezoneFromUTC import determine_correct_format
+
+
+def test_convert_UTC_Timezone_command():
+    """
+        Given -
+           UTC time, timezone, format
+        When -
+            using the converting function
+        Then -
+            the function will return the converted time, from UTC to local timezone
+    """
+    timezone = "US/Eastern"
+    value = "2023-01-04 18:14:18"
+    format = "%Y-%m-%d %H:%M:%S"
+    time_as_datetime_type = determine_correct_format(time=value, fmt=format)
```

Comment:
In general it will be better practice to just use the actual result of this function for this test and not running it, otherwise your test does not check only one thing, meaning that if it fails you need to investigate which function it failed for instead of understanding that directly from the test.

Reply:
agree.
 in this specifice case its not so simple, becous it needs a datetime obj, so i can create one here using "datetime.datetime" and mock the "determine_correct_format" func, but thats equel to what i did becus determine_correct_format uses "datetime.datetime".